### ISSUE TITLE
Prepare for release

### ISSUE THREAD

**Issue by wormeyman** [futtta/async-javascript#20]
I am thinking:

1. we bump up to 3.18.04.23 as we have made so many changes this might break some things
1. I send another PR with updates to the changelog
1. anything else?

**Comment by futtta** [comment]
OK for 1 & 2

for 3; interested in joining me doing support on the wordpress.org forum? in that case add your wordpress.org username ot the readme.txt contributors line and/ or I'll add you as a "support rep"? Not too many questions up until now, but always nice to have a backup (esp. since a lot of my time goes into autoptimize) ... :-)

**Comment by wormeyman** [comment]
Yes, I would love to help.

**Comment by futtta** [comment]
On 2nd thought I'm not sure we should do the major version bump; as you write in the changelog the functionality does not change and the rewrite was not _that_ big? I can be convinced, but I'll need arguments :sunglasses:

**Comment by wormeyman** [comment]
Sure we can leave it on two not a big deal. You have a good point!

**Comment by futtta** [comment]
just to make sure; can you test the wizard (again) to make sure this works for you?

**Comment by wormeyman** [comment]
Yes you bet. It might not be until Monday.

**Comment by futtta** [comment]
monday's OK mate, enjoy the weekend :-)

**Comment by futtta** [comment]
ping, monday's here ;-)

**Comment by wormeyman** [comment]
Still works for me!

**Comment by futtta** [comment]
![image](https://user-images.githubusercontent.com/181297/39172546-6054dbdc-47a2-11e8-8e85-c4be30c51fd7.png)

:-)

**Comment by futtta** [comment]
@wormeyman so [there's a support question](https://wordpress.org/support/topic/wizard-sticks-at-step-2/), would you care to have a go at it?

(posting here by lack of a better way of communicating, maybe we should exchange mail addresses or use wordpress.org's slack?)

**Comment by wormeyman** [comment]
Yeah, WhatsApp or email work well. I can reinstall slack if I have too. 😋

**Comment by futtta** [comment]
let's do mail; I'm frank-at-optimizingmatters-dot-com :-)
